Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10
```

Comment:
As node.js 12.x is the current LTS I think it would be good to use `FROM node:12` here (and I run 12.x locally, so I can confirm the exerciser is good on that version)